azure/service.ts

```diff
@@ -102,7 +102,7 @@ function getEndpointHelper(
 // objets as we're building them up before making the final Group.
 
 interface AzureContainer {
-    command?: pulumi.Input<string>;
+    commands?: pulumi.Input<string[]>;
```

Comment:
terraform supports both of these.  However 'command' is deprecated and 'commands' is recommended.  That's useful given that that's also how our internal 'command' value is stored.